perf_test/Benchmark_Context.hpp

```diff
@@ -89,10 +90,30 @@ inline void add_version_info() {
   }
 }
 
+inline void add_env_info() {
+  auto num_threads = std::getenv("OMP_NUM_THREADS");
+  if (num_threads) {
+    benchmark::AddCustomContext("OMP_NUM_THREADS", num_threads);
+  }
+  auto dynamic = std::getenv("OMP_DYNAMIC");
+  if (dynamic) {
+    benchmark::AddCustomContext("OMP_DYNAMIC", dynamic);
+  }
+  auto proc_bind = std::getenv("OMP_PROC_BIND");
+  if (proc_bind) {
+    benchmark::AddCustomContext("OMP_PROC_BIND", proc_bind);
+  }
+  auto places = std::getenv("OMP_PLACES");
+  if (places) {
+    benchmark::AddCustomContext("OMP_PLACES", places);
+  }
```

Comment:
This could be protected with `#if defined(KOKKOS_ENABLE_OPENMP)`, but it's pretty harmless even if a different backend is used.